pkgs/applications/editors/jupyter-kernels/octave/default.nix

```diff
@@ -51,13 +51,14 @@ rec {
 
       src = octave.src;
 
-      buildInputs = [ imagemagick ];
+      nativeBuildInputs = [ imagemagick ];
+      strictDeps = true;
 
       dontConfigure = true;
       dontInstall = true;
 
       buildPhase = ''
-        convert ./libgui/src/icons/octave/128x128/logo.png -resize ${size}x${size} $out
```

Comment:
do we want to add runHook {pre,post}Build while we're at it?

Reply:
Might be better to just convert this to `runCommand` instead?